game/main.py

```diff
@@ -107,7 +107,7 @@ def reveal():
 def newgame():
     if len(Game.catchphrases) < 1:
         print("No more catchphrases")
-        return redirect("/", code=301)
+        return redirect("/", code=302)
```

Comment:
302 is perminant redirect, 301 is temporary. This will break the new-game functionality as it will never be able to perform a newgame after the 1st time it reaches this line (you will need to clear browser cache)